README.md

```diff
@@ -35,5 +35,9 @@ vim.keymap.set('n', '<c-j>', require('tree-climber').swap_next, keyopts)
 
 Each function optionally accepts a table with a configuration, for example, `goto_next({ skip_comments = true})`.
 
-The only available option so far is:
+The available options so far are:
 * `skip_comments` (boolean) - ignore comment nodes as if they were not there at all (default: false)
+* `highlight` (boolean) - When moving using the `goto_*()` functions, briefly highlight the new node (default: false)
+* `timeout` (number) - When highlighting, the time in ms before highlight is cleared (default: 150)
+* `on_macro` (boolean) - When highlighting, highlight when executing a macro (default: false)
```

Comment:
I think this would be better not "when highlighting, ...", but rather "whether to highlight nodes when executing a macro", or something like that